### PR TITLE
Fix for allegro.pl

### PR DESCRIPTION
**Pull request by ejbtrd**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -822,6 +822,7 @@ allegro.pl
 INVERT
 i[title="Smart!"]
 [alt*="smart" i]
+[alt*="Smart" img]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:825]: This is not correct CSS syntax, what is this supposed to fix?

**Review by Gusted — COMMENTED** [review]
CC @FadeMind, given you have done multiple fixes on this domain.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: `[alt*="smart" i]`



Don't touch this if works fine 💯

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: Also ` [alt*="Smart" img] ` break all and in result logo is not inverted.  @Gusted

Comment (ejbtrd) [reply src/config/dynamic-theme-fixes.config:825]: uh i mean i am not that familliar with darkreader to be honest, but that fixed this banner for me

image

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @ejbtrd not all carusell images are damaged

I purged all this stuffs via Alleblock Adguard rules. This is ad, nothing more. 
Exclude can be done via select valid selector for not invert.

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @ejbtrd @Gusted Please remove your suggestion and try
```
ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
```

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @ejbtrd second 
```
ul[data-role="image-tiles"] > li[data-analytics-view-opbox-custom-destination-url="https://allegro.pl/kampania/smartweek"]
```

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: CC @Gusted ping

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:825]:
```suggestion
ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
ul[data-role="image-tiles"] > li[data-analytics-view-opbox-custom-destination-url="https://allegro.pl/kampania/smartweek"]
```

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @ejbtrd where is banner with Allegro Lokanie? I cannot find it. 🤔

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @Gusted in a free time I will prepare new allegro fixes - some fonts are to dimmed.

Comment (ejbtrd) [reply src/config/dynamic-theme-fixes.config:825]: > @ejbtrd @Gusted Please remove your suggestion and try
> 
> ```
> ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
> ```

This selector fixes Smart Week banner on main page, however the whole issue started from https://allegro.pl/smart.

Right now i have also noticed that many more images are incorrectly inverted:

image

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @Gusted @ejbtrd please wait 💯

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: ping

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: CC @Gusted ping

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:825]: So:
```suggestion
ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
ul[data-role="image-tiles"] > li[data-analytics-view-opbox-custom-destination-url="https://allegro.pl/kampania/smartweek"]
a[data-analytics-click-value="Smart lokalnie"]
```

Would be correct?

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:825]: @Gusted for this "issues" for a while yes. 
Allegro changes many times banners codes, sadly not universal catch work here.

**Review by Gusted — APPROVED** [review]

**Pull request merged**